yasa/hypno.py

```diff
@@ -74,6 +74,10 @@ class Hypnogram:
     scorer : str
         An optional string indicating the scorer name. If specified, this will be set as the name
         of the :py:class:`pandas.Series`, otherwise the name will be set to "Stage".
+    proba : :py:class:`pandas.DataFrame`
+        An optional dataframe with the probability of each sleep stage for each epoch in hypnogram.
+        Each row must sum to 1. This is automatically included if the hypnogram is created with
```

Comment:
This might not be the best place to comment this, but regarding the `proba` that is automatically returned from `SleepStaging`, should this not have a `Hynogram.scorer` attached to it? Maybe just `"YASA"` or even something with the version number like `"YASA-vX.X"`? Or I'm not sure if you have a separate name and/or version control for the underlying sleep stager.

But in any case, I think the usefulness of the `scorer` attribute is that it allows for convenient comparisons, especially if someone is saving their data and looking back at it after a new stager comes out. Note also that the `evaluation` module plotting functions often take advantage of this attribute for plotting and dataframe organization, so using it in the returned dataframe might help out in those instances too.

Reply:
Hmm, I like the idea but what would be the best way to attach such an attribute to a pandas DataFrame? In any cases, I think this can be done in a separate PR.

Reply:
Agreed, this is overly complicated for little gain in the immediate future. We'll see if it comes up again later.